bin/webpack.js

```diff
@@ -1,21 +1,71 @@
 #!/usr/bin/env node
 
+const cp = require("child_process");
+const inquirer = require("inquirer");
```

Comment:
Could you move these two requires into the runCommand function resp. the if block.
So we don't pay the startup overhead when webpack-cli is installed.

Reply:
Ok, commiting that!